tests/scripts/github_docs_comment.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import json
+import argparse
+from urllib import error
+
+from git_utils import git, GitHubRepo, parse_remote
+
+
+def build_docs_url(base_url_docs, pr_number):
+    return base_url_docs + "/tvm/PR-" + str(pr_number) + "/docs/index.html"
+
+
+if __name__ == "__main__":
+    help = "Add comment with link to docs"
+    parser = argparse.ArgumentParser(description=help)
+    parser.add_argument("--remote", default="origin", help="ssh remote to parse")
+    parser.add_argument("--base-url-docs", default="https://d3f1x5ne0bf47p.cloudfront.net")
+    parser.add_argument(
+        "--dry-run",
+        action="store_true",
+        default=False,
+        help="run but don't send any request to GitHub",
+    )
+    args = parser.parse_args()
+
+    remote = git(["config", "--get", f"remote.{args.remote}.url"])
+    user, repo = parse_remote(remote)
+
+    pr = json.loads(os.environ["PR"])
+    number = pr["number"]
+
+    docs_url = build_docs_url(args.base_url_docs, number)
+
+    url = f"/issues/{number}/comments"
+    body = f"Built docs can be found [here]({docs_url})."
+    if not args.dry_run:
+        github = GitHubRepo(token=os.environ["GITHUB_TOKEN"], user=user, repo=repo)
+        try:
+            github.post(url, {"body": body})
+        except error.HTTPError as e:
+            print(f"Failed to add docs comment {docs_url}: {e}")
```

Comment:
Can you use `logging` to get a full traceback here (and replace the other `print`s as well?) the other Python scripts in `tests/scripts` have some example usages (just call `cmd_utils.init_log()` and then use `logging.info`)
```suggestion
            logging.exception(f"Failed to add docs comment {docs_url}: {e}")
```